src/hardware/input/mouseif_dos_driver.cpp

```diff
@@ -49,6 +50,14 @@ CHECK_NARROWING();
 // - https://www.stanislavs.org/helppc/int_33.html
 // - http://www2.ift.ulaval.ca/~marchand/ift17583/dosints.pdf
 // - https://github.com/FDOS/mouse/blob/master/int33.lst
+// - https://www.fysnet.net/faq.htm
+
+// Versions are stored in BCD code - 0x09 = version 9, 0x10 = version 10, etc.
+static constexpr uint8_t driver_version_major = 0x08;
+static constexpr uint8_t driver_version_minor = 0x05;
+
+static const std::string str_version_prefix = "version ";
```

Comment:
The space is intentional? Maybe put a comment there that the trailing space is there for a reason (one could mistake it for a typo and remove it).

Reply:
I got rid of this helper string, it is used in one place nevertheless.